packages/functional-tests/tests/syncV3/syncV3EmailFirst.spec.ts

```diff
@@ -4,27 +4,23 @@
 
 import { expect, test, PASSWORD } from '../../lib/fixtures/standard';
 
-let email;
-
 test.describe.configure({ mode: 'parallel' });
 
 test.describe('Firefox Desktop Sync v3 email first', () => {
+  test.use({ emailOptions: [{ prefix: 'sync{id}', PASSWORD }] });
   test.beforeEach(async ({ pages: { login } }) => {
     test.slow();
-    email = login.createEmail('sync{id}');
   });
 
-  // TODO: is this something we want to support once index/email-first is converted to React?
   test('open directly to /signup page, refresh on the /signup page', async ({
+    emails,
     pages: { configPage },
     target,
     syncBrowserPages: { page, login },
   }) => {
+    const [email] = emails;
     const config = await configPage.getConfig();
-    test.skip(
-      config.showReactApp.signUpRoutes === true,
-      'Not currently supported in React Signup'
-    );
+    test.fixme(true, 'test to be supported after FXA-8973 is done');
```

Comment:
I think this could stay a skip, or at least leave the condition in the fixme.

```suggestion
    test.fixme(config.showReactApp.signUpRoutes === true, 'Not currently supported in React Signup FXA-8973');
```

Reply:
Yeah I think it should still be fixme() but I can change the comment.